.github/workflows/olm_scorecard.yml

```diff
@@ -1,28 +0,0 @@
-name: Operator Scorecard Test
-
-on: [push, pull_request]
-
-jobs:
-  scorecard-check:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Check out code
-      uses: actions/checkout@v2
-
-    - name: Setup and start KinD cluster
-      uses: ./.github/actions/kind
-
-    - name: Build bundle
-      run: make bundle-build
-
-    - name: Install yq
-      run: |
-        sudo wget -O /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.6.1/yq_linux_amd64
-        sudo chmod +x /usr/bin/yq
-
-    - name: Modify scorecard config
-      run: |
-        yq e 'del(.stages[].tests[] | select(.labels.test == "olm-crds-have-resources-test"))' -i bundle/tests/scorecard/config.yaml
-
-    - name: Run Operator SDK Scorecard
-      run: make scorecard-bundle
```

Comment:
I think we may remove the scorecard-bundle in the Makefile.

Reply:
Removed